streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -199,7 +195,16 @@ public StateStoreSupplier build() {
          * @return the interface used to specify the remaining key-value store options; never null
          */
         public KeyValueFactory<K, Long> withLongValues() {
-            return withValues(new LongSerializer(), new LongDeserializer());
+            return withValues(Serdes.Long());
+        }
+
+        /**
+         * Use {@link Double} values.
+         *
+         * @return the interface used to specify the remaining key-value store options; never null
+         */
+        public KeyValueFactory<K, Double> withDoubleValues() {
```

Comment:
Should we also add `withByteBufferValues` and `withByteBufferKeys` because we introduced `ByteBufferSerde` in the latest update to this PR?